src/shared/test/user-agent-test.js

```diff
@@ -19,6 +19,14 @@ describe('shared/user-agent', () => {
     });
   });
 
+  describe('isTouchDevice', () => {
+    it('calls `window.matchMedia` with the query string "(pointer: coarse)"', () => {
+      const matchMedia = sinon.spy(window, 'matchMedia');
```

Comment:
This spy is not reset anywhere. I would suggest to stub `matchMedia` in a `beforeEach` and reset in an `afterEach` block.

Reply:
oops, good catch